.travis.yml

```diff
@@ -10,9 +10,14 @@ cache:
   directories:
     - .eslint-cache
     - .jest-cache
+    - ~/.cache
 script:
   - yarn test-coverage
   - yarn build
+  # Integration test, only in tags, master and develop
+  - if [[ $TRAVIS_TAG != "" || $TRAVIS_BRANCH = master || $TRAVIS_BRANCH = develop ]]; then
+      yarn test:integration;
+    fi
```

Comment:
Why not on PRs?

I would expect to have the integration tests to run on every PR change, as they are supposed to assure no changes are breaking the code.

It's true that this stops a failed change to be deployed into develop/master/tags, but doesn't prevent a breaking change from being merging into those branches.